### ISSUE TITLE
Security advisory affecting this module (NSP) nodesecurity.io

### ISSUE THREAD

**Issue by guerrerocarlos** [sendgrid/sendgrid-nodejs#672]
#### Issue Summary

Security advisory affecting this module:
https://nodesecurity.io/advisories/566

#### Steps to Reproduce

- Install @sendgrid/mail on a project using npm or yarn
- execute `nsp check`

NSP Output:

```
vulnerabilities found
@sendgrid/mail@6.2.1 > @sendgrid/client@6.2.1 > request@2.83.0 > hawk@6.0.2 > hoek@4.2.0
```

#### Technical details:

* sendgrid-nodejs Version: @sendgrid/mail@6.2.1
* Node.js Version: v8.7.0

**Comment by thinkingserious** [comment]
Thanks for the heads up @guerrerocarlos!

**Comment by Berkmann18** [comment]
This seems fixed on the current version (6.3.1).
